UserService/UserService.DAL/UoW/UnitOfWork.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UserService.DAL.Interfaces;
+
+namespace UserService.DAL.UoW
+{
+    public class UnitOfWork(ApplicationDbContext context) : IUnitOfWork
+    {
+        public async Task<int> SaveChangesAsync(CancellationToken cancellationToken)
```

Comment:
unit of work must contain repositories and all work with repositories need to be implemented with UoW